src/SignalR/clients/csharp/Http.Connections.Client/src/HttpConnection.cs

```diff
@@ -365,6 +374,13 @@ private async Task SelectAndStartTransport(TransferFormat transferFormat, Cancel
                         continue;
                     }
 
+                    if (transportType == HttpTransportType.ServerSentEvents && _isRunningInBrowser)
+                    {
+                        Log.ServerSentEventsNotSupportedByBrowser(_logger);
+                        transportExceptions.Add(new TransportFailedException("ServerSentEvents", "The transport is not supported in the browser."));
```

Comment:
Why is that? Is it because the http client isn't fetch based?

Reply:
From @BrennanConroy on the other PR:

> Was thinking about this over the weekend and realized we need to do something about SSE.

Right now it doesn't work because we use an HttpClient GET with "text/event-stream" Accept header. Which doesn't work in the browser, we use the EventSource API in the Typescript client to do a streaming response.

We might just have to disable SSE when running the .NET client in the browser.

Reply:
I believe @brennanconroy tried it and it didn’t work. We can always make adjustments and reenable. Plus, we are considering deprecating SSE in general.